src/main/java/net/earthcomputer/clientcommands/mixin/MixinGameRenderer.java

```diff
@@ -22,11 +23,16 @@
     @Inject(method = "renderWorld", at = @At(value = "INVOKE_STRING", target = "Lnet/minecraft/util/profiler/Profiler;swap(Ljava/lang/String;)V", args = {"ldc=hand"}))
     private void renderWorldHand(float delta, long time, MatrixStack matrixStack, CallbackInfo ci) {
         matrixStack.push();
+
+        //Render lines through everything
+        RenderSystem.clear(256, MinecraftClient.IS_SYSTEM_MAC);
```

Comment:
Not sure what 256 is, can you check if this is `GL11.GL_DEPTH_BUFFER_BIT`? Always used named constants in GL calls rather than magic constants like this.